src/unix/linux_like/linux/mod.rs

```diff
@@ -504,6 +508,43 @@ s! {
         pub ee_info: u32,
         pub ee_data: u32,
     }
+
+    // linux/can.h
+    #[repr(align(8))]
+    pub struct can_frame {
+        pub can_id: canid_t,
+        pub can_dlc: u8,
+        pub __pad: u8,
+        pub __res0: u8,
+        pub __res1: u8,
```

Comment:
They don't have to be public?
```suggestion
        __pad: u8,
        __res0: u8,
        __res1: u8,
```